torch/_inductor/mkldnn_ir.py

```diff
@@ -238,10 +238,8 @@ def __init__(
             inputs,
             constant_args,
             None,
-            python_kernel_name="torch.ops.mkldnn._convolution_pointwise",
-            cpp_kernel_name="mkldnn::_convolution_pointwise",
+            op_overload=torch.ops.mkldnn._convolution_pointwise.default,
```

Comment:
Does that mean `self.python_kernel_name` and `self.cpp_kernel_name` should be generated by `self.op_overload`?

Reply:
Yes, they will be generated in the base class ExternKernel.